src/styles/index.css

```diff
@@ -36,8 +37,8 @@ h1 {
 
 @media (prefers-color-scheme: light) {
```

Comment:
Let's delete this media query for now. We can worry about responsive design later.